README.md

```diff
@@ -15,11 +15,16 @@ For IOOS python users, the
 is the easiest way to get custom packages needed to run the IOOS system tests.
 Using Anaconda users can `conda install`, which installs not only binary
 packages for their platform, but the binary libraries they depend on.
-So easier than `pip install` and more powerful than python wheels.
-System-level installation of libraries and admin privileges are not requires.
-Check out
+So easier than `pip install` and, thanks to Binary Relocation, more powerful
+than python wheels.  System-level installation of libraries and admin
+privileges are not requires. Check out
 [Travis Oliphant's blog piece](http://technicaldiscovery.blogspot.com/2013/12/why-i-promote-conda.html) for more info.
 
-The recipes here are automatically tested using
-[Travis-CI](https://travis-ci.org/ioos/conda-recipes) with the assistance of [Obvious-CI](https://github.com/pelson/Obvious-CI/
-https://github.com/ioos/conda-recipes/blob/master/.travis.yml#L14-L31). 
+The recipes here are automatically built using
+[Travis-CI](https://travis-ci.org/ioos/conda-recipes) with the assistance of
+[Obvious-CI](https://github.com/pelson/Obvious-CI/https://github.com/ioos/conda-recipes/blob/master/.travis.yml#L14-L31).
+
+**CAVEAT:**  Windows users must be aware that the packages here are not fully
```

Comment:
Do we really need this caveat about testing?  I can test on windows, no? 
 I'd just end with "raise and issue" and get rid of that upstream link.